### DISCUSSION TITLE
Support for Kobo Aura H2O (N250)?

### DISCUSSION THREAD

**Discussion by foo-git** [Quill-OS/quill#1] [General]
Hi,

I really see the potential having a "real" OS running on a Kobo device for home automation stuff, but I have a rather old device from 2015, a Kobo Aura H2O (Model N250).
Is Inkbox/Kobox able to run on this device, or is there an underlying technical issue?
I'm willing to fund some euros if it's technically feasible, and also to help with debugging with my device.

**Comment by tux-linux** [comment]
You are lucky! And your device is less "old" than most of the currently supported devices by InkBox (actually, it unofficialy runs on a 2019 Libra H2O, without an internal SD card, it's dangerous, but it works)
Theoritically, it could. I'd be happy if you came in the Discord server there so we could chat a bit: https://discord.com/invite/uSWtWbY23m
Also, feel free to join the Mobileread unofficial one: https://discord.com/invite/FTAMN8wP6d
Thanks for wanting to contribute, I really appreciate it!
